runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8269,6 +8269,29 @@ done:;
 			/* Get MethodHandle for this operation from the VarHandles handleTable */
 			j9object_t handleTable = J9VMJAVALANGINVOKEVARHANDLE_HANDLETABLE(_currentThread, varHandle);
 			j9object_t methodHandle = J9JAVAARRAYOFOBJECT_LOAD(_currentThread, handleTable, operation);
+
+			if (NULL == methodHandle) {
+				prepareForExceptionThrow(_currentThread);
+
+				const char *exceptionClassName = "java/lang/UnsupportedOperationException";
+				J9Class *exceptionClass = internalFindClassUTF8(
+						_currentThread,
+						(U_8 *)exceptionClassName,
+						strlen(exceptionClassName),
+						_currentThread->javaVM->systemClassLoader,
+						J9_FINDCLASS_FLAG_THROW_ON_FAIL);
+
+				j9object_t exception = _objectAllocate.inlineAllocateObject(_currentThread, exceptionClass);
+				if (NULL == exception) {
+					rc = THROW_HEAP_OOM;
+					goto done;
+				}
+
+				VM_VMHelpers::setExceptionPending(_currentThread, exception);
+				rc = GOTO_THROW_CURRENT_EXCEPTION;
+				goto done;
+			}
```

Comment:
@DanHeidinga New code to address the segfault in the four `View-(BE|LE)-(On|Off)Heap_0` testcases. I couldn't build a method type frame since no method type is available. So, I ended up moving `prepareForExceptionThrow` before looking up the exception class.

New code is included in the following commit: 
> Handle NULL VarHandle.handleTable entries in (Bytecode|MH)Interpreter